Description: Fix font search path for fontdir /usr/share/wine/fonts.
Author: Jens Reyer <jre.winesim@gmail.com>
Bug: https://bugs.winehq.org/show_bug.cgi?id=40312
Bug-Debian: https://bugs.debian.org/814844

--- a/dlls/gdi32/freetype.c
+++ b/dlls/gdi32/freetype.c
@@ -2980,9 +2980,9 @@ static char *get_data_dir_path( LPCWSTR
     {
         INT len = WideCharToMultiByte(CP_UNIXCP, 0, file, -1, NULL, 0, NULL, NULL);
 
-        unix_name = HeapAlloc(GetProcessHeap(), 0, strlen(data_dir) + len + sizeof("/fonts/"));
+        unix_name = HeapAlloc(GetProcessHeap(), 0, strlen(data_dir) + len + sizeof("/../../wine/fonts/"));
         strcpy(unix_name, data_dir);
-        strcat(unix_name, "/fonts/");
+        strcat(unix_name, "/../../wine/fonts/");
 
         WideCharToMultiByte(CP_UNIXCP, 0, file, -1, unix_name + strlen(unix_name), len, NULL, NULL);
     }
